youtube_dl/extractor/vlive.py

```diff
@@ -61,16 +47,14 @@ def _real_extract(self, url):
         elif status == 'VOD_ON_AIR' or status == 'BIG_EVENT_INTRO':
             if long_video_id and key:
                 return self._replay(video_id, webpage, long_video_id, key)
-            elif is_live:
-                status = 'LIVE_END'
             else:
                 status = 'COMING_SOON'
 
         if status == 'LIVE_END':
             raise ExtractorError('Uploading for replay. Please wait...',
                                  expected=True)
         elif status == 'COMING_SOON':
-            raise ExtractorError('Coming soon! %s' % air_start, expected=True)
+            raise ExtractorError('Coming soon!', expected=True)
```

Comment:
Extra request and 12 lines of code (including timezone offset cookie) isn't worth it in my opinion.